lib/3scale/tasks/connectivity.rake

```diff
@@ -0,0 +1,29 @@
+namespace :connectivity do
+  desc 'Check connectivity of Redis Storage'
+  task :redis_storage_check do
+    begin
+      redis_instance = ThreeScale::Backend::Storage.instance
+      redis_instance.ping
+      puts "Connection to Redis Storage performed successfully"
```

Comment:
I've added a print showing that the connection is successful in case it can ping the Redis destination. This also has been applied to redis queue. It is useful for debugging purposes